modules/social_features/social_landing_page/templates/node--featured.html.twig

```diff
@@ -156,11 +156,13 @@
       {% endif %}
 
       {% block card_link %}
-        <div class="card__link">
-          <a href="{{ url }}" rel="bookmark">{{ 'Read more'|t }}
-            <span class="visually-hidden">{% trans %}about {{ label }}{% endtrans %} </span>
-          </a>
-        </div>
+        {% if not hide_card_link %}
+          <div class="card__link">
+            <a href="{{ url }}" rel="bookmark">{{ 'Read more'|t }}
+              <span class="visually-hidden">{% trans %}about {{ label }}{% endtrans %} </span>
+            </a>
+          </div>
+        {% endif %}
```

Comment:
Is there a reason we don't just generate this link in a preprocess hook and render it as {{ read_more }} ? Then if a preprocess hook wants to change it or hide it that can be done without introducing a special logic variable.

Reply:
This is done to make it easier to add new classes, wrappers, etc. by front-end developers

Reply:
I think it was just done like this before, and within this story, we didn't refactor templates themselves.
Oleksandr just adds a condition, which allows hiding read more link, which is relevant for the Dashboard.